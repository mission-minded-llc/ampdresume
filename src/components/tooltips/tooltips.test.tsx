import "@testing-library/jest-dom";
import React from "react";
import { render } from "@testing-library/react";
import { TooltipTotalYears } from "./";

describe("TooltipTotalYears", () => {
  it("matches snapshot", () => {
    const { container } = render(<TooltipTotalYears />);
    expect(container).toMatchSnapshot();
  });
});
