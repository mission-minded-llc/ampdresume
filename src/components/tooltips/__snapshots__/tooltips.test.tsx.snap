// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`TooltipTotalYears matches snapshot 1`] = `
<div>
  <p>
    If you enter the 
    <strong>
      year started
    </strong>
    , the
     
    <strong>
      total years
    </strong>
     will be calculated for you automatically.
  </p>
  <p>
    If you enter a 
    <strong>
      total years
    </strong>
     value, it will override the calculated value.
  </p>
</div>
`;
